jujugui/static/gui/src/app/components/deployment-flow/credential/add/add.js

```diff
@@ -166,6 +166,9 @@ class DeploymentCredentialAdd extends React.Component {
     if (!info || !info.forms) {
       return;
     }
+    if (!info.forms[this.state.authType]) {
```

Comment:
Was this a bug?

Reply:
It appeared to be, yes. I don't think we ever hit the bug due to the props we had, but I suspect we would have hit it down the track depending on the cloud configs.